lib/SftpClient.js

```diff
@@ -0,0 +1,49 @@
+const SFTP = require('sftp-promises')
+const FileParser = require('ftp/lib/parser')
+
+class SftpClient {
+  constructor ({ host, port = 21, user, password }) {
+    this.host = host
+    this.port = port
+    this.user = user
+    this.password = password
+    this.client = new SFTP()
```

Comment:
Also add `this.session = null` at the end. Initializing all member variables in the constructor gives a better overview of all possible variables, makes debugging easier and also helps the VM optimizing.